src/util/util.shared.ts

```diff
@@ -171,14 +181,14 @@ export const Utils = {
       return `${url}${Constant.EOS_PREFIX}get_info`
     }
 
-    if (isUtxoBasedNetwork(network) || isEvmBasedNetwork(network) || isTronNetwork(network)) {
+    if (isUtxoBasedNetwork(network) || isEvmBasedNetwork(network) || isTronNetwork(network) || isSolanaNetwork(network)) {
```

Comment:
It would be nice to extract this huge if into a separate function at this point, it will only keep getting bigger.

Reply:
And how you would name it?